src/cache_cf.cc

```diff
@@ -2950,7 +2950,7 @@ parse_TokenOrQuotedString(char **var)
 static void
 dump_time_t(StoreEntry * entry, const char *name, time_t var)
 {
-    storeAppendPrintf(entry, "%s %d seconds\n", name, (int) var);
+    storeAppendPrintf(entry, "%s %ld seconds\n", name, static_cast<long>(var));
```

Comment:
I am not excited about this change because it makes the problem more difficult to see rather than fixing it: `long int` is [guaranteed](https://en.cppreference.com/w/cpp/language/types) to have 32 bits, not 64 bits as this fix (description) may want us to believe. I made the same mistake in commit 5ef3cbb, but I did not have Coverity results to help me see the light.

Want a proper minimal fix? Cast to and printf() int64_t.

Want to stop kicking the can down the road? Convert dump_time_t() to use PackableStream instead. See SBufStatsAction::dump(), dump_refreshpattern(), Mem::Stats(), and similar functions for examples.